src/frontend/XAPI/XAPIStatement.spec.ts

```diff
@@ -61,7 +61,6 @@ describe('XAPIStatement', () => {
         'https://w3id.org/xapi/video/extensions/session-id': 'abcd',
       });
       expect(body).toHaveProperty('id');
-      expect(body).toHaveProperty('timestamp');
```

Comment:
Shouldn't your statement still have a timestamp (now coming from the backend)?

Reply:
Nop the timestamp is added once this statement received by the backend:

front -> back -> LRS